src/Pim/Component/Catalog/Query/Filter/FieldFilterHelper.php

```diff
@@ -74,6 +74,22 @@ public static function checkArray($field, $value, $className)
     }
 
     /**
+     * Check if value is a string
+     *
+     * @param string $field
+     * @param mixed  $value
+     * @param string $className
+     *
+     * @throws InvalidPropertyTypeException
+     */
+    public static function checkString($field, $value, $className)
+    {
+        if (!is_string($value) && null !== $value) {
```

Comment:
Yep, but is it the responsability of a checkString method to check that ? I find it a little bit weird :-/ or an additionnel argue as "allowNullable" which will precise what the method really do ;-)